apps/hash-api/package.json

```diff
@@ -44,6 +44,7 @@
     "@ory/kratos-client": "0.13.1",
     "@rudderstack/rudder-sdk-node": "2.0.3",
     "@snowplow/node-tracker": "3.3.1",
+    "@temporalio/client": "1.8.1",
```

Comment:
This is non-blocking, but we should update `@temporalio/worker` to `1.8.1` in a future PR – the `@temporalio/*` packages are part of an SDK and seem to get the same version bumps. Aligning them is probably best to avoid type mismatches or weird behaviour